code/modules/wiremod/components/math/arithmetic.dm

```diff
@@ -16,17 +16,16 @@
 
 	circuit_flags = CIRCUIT_FLAG_INPUT_SIGNAL|CIRCUIT_FLAG_OUTPUT_SIGNAL
 
-GLOBAL_LIST_INIT(comp_arithmetic_options, list(
-	COMP_ARITHMETIC_ADD,
-	COMP_ARITHMETIC_SUBTRACT,
-	COMP_ARITHMETIC_MULTIPLY,
-	COMP_ARITHMETIC_DIVIDE,
-	COMP_ARITHMETIC_MIN,
-	COMP_ARITHMETIC_MAX,
-))
-
 /obj/item/circuit_component/arithmetic/Initialize()
-	options = GLOB.comp_arithmetic_options
+	var/static/component_options = list(
+		COMP_ARITHMETIC_ADD,
+		COMP_ARITHMETIC_SUBTRACT,
+		COMP_ARITHMETIC_MULTIPLY,
+		COMP_ARITHMETIC_DIVIDE,
+		COMP_ARITHMETIC_MIN,
+		COMP_ARITHMETIC_MAX,
+	)
+	options = component_options
```

Comment:
You can put this in the object definition directly.

Reply:
if you're talking about the "options" variable, I did it like this so that you don't have a bunch of duplicate lists for each component that is made.